contracts/utils/DAOFactory.sol

```diff
@@ -16,11 +16,23 @@ contract DAOFactory is Initializable {
     event NewOrg (address indexed _avatar);
     event InitialSchemesSet (address indexed _avatar);
     event SchemeInstance(address indexed _scheme, string indexed _name);
+    /**
+    * @dev Emitted when a new proxy is created.
+    * @param _proxy Address of the created proxy.
+    * @param _version of the created proxy.
+    */
+    event ProxyCreated(address _proxy, uint64[3] _version);
+
+    struct Locks {
+        address sender;
+        uint64[3] packageVersion;
+    }
 
-    mapping(address=>address) public locks;
+    mapping(address=>Locks) public locks;
     App public app;
     string public constant PACKAGE_NAME = "DAOstack";
     DAOTracker private daoTracker;
+    uint64[3] private packageVersion;
```

Comment:
Why is that a storage variable. Wouldn't it be cheaper (and better) to just pass it to the `_forgeOrg` function?

Reply:
it will be cheaper , though this is here due to  "stack is too deep" issue .